Plugin.php

```diff
@@ -57,7 +57,12 @@ public function initialize()
 	$customizer['login_btn_shadow'] = $this->configModel->get('login_btn_shadow', '0');
 	$customizer['login_btn_border'] = $this->configModel->get('login_btn_border', '0');
 	$customizer['login_btn_width'] = $this->configModel->get('login_btn_width', '95');
-	    
+	//Personalized style 
```

Comment:
This shouldn't be needed, if we go this route.

The login page is different, because there are no Authorization Permissions established before a user logs in. Meaning, `$this->configModel` is not accessible. So the values had to be passed to a global variable in order to be accessible.